packages/gasket-plugin-manifest/lib/middleware.js

```diff
@@ -0,0 +1,30 @@
+const escapeRegex = require('escape-string-regexp');
+const { gatherManifestData } = require('./utils');
+
+/**
+ * Add some middleware to gather manifest details for certain endpoints
+ *
+ * @param {Gasket} gasket - The gasket API
+ * @returns {function} Express middleware to apply
+ */
+function handler(gasket) {
+  const { serviceWorker: { url: swUrl = '' } = {} } = gasket.config;
+
+  const endpoints = [/manifest\.json/];
+  if (swUrl) endpoints.push(new RegExp(escapeRegex(swUrl)));
+
+  return async function manifestMiddleware(req, res, next) {
```

Comment:
We should only add a middleware if no `staticOutput`.

You can exit early here:
```js
if(staticOutput) return;
```